server/bleep/src/webserver/file.rs

```diff
@@ -2,12 +2,11 @@ use std::sync::Arc;
 
 use axum::{
     extract::{Path, Query},
-    response::{IntoResponse, Result},
+    response::IntoResponse,
     Extension,
 };
-use reqwest::StatusCode;
 
-use super::{json, prelude::Indexes, EndpointError, ErrorKind};
+use super::{json, prelude::Indexes, Error};
```

Comment:
would you mind cleaning up all the imports to make use of `prelude`? this could be `use crate::webserver::prelude::*`.